encord/objects/ontology_object_instance.py

```diff
@@ -198,7 +201,7 @@ def set_answer(
                 "The answer to this attribute was already set. Set `overwrite` to `True` if you want to"
                 "overwrite an existing answer to an attribute."
             )
-
+        static_answer.is_manual_annotation = manual_annotation
```

Comment:
I wonder if it would be better to add this flat to the Answer.set() method signature, so setting answer and stating is it manual annotation or not would be a single operation? 
WDYT?